### PR TITLE
[MT-4670] Fix - Placeholder should not be visible for white-space-only text blocks

### PR DESCRIPTION
**Pull request by OskarGasiorowski**
https://user-images.githubusercontent.com/41072665/165494467-56a0b84a-628f-46a0-b9f4-5f96fba3e999.mp4

### CODE REVIEW

File: packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx [hunk 1]
```diff
@@ -196,3 +197,7 @@ export function FloatingAddMenu<Action>({
         </FloatingContainer.Container>
     );
 }
+
+function hasOnlySpaces(text: string) {
```
Comment (OskarGasiorowski) [inline packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx:201]: @e1himself @aspirisen If you have better place for that let me know :)

File: packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx [hunk 2]
```diff
@@ -164,7 +165,7 @@ export function FloatingAddMenu<Action>({
                     />
                 )}
             </TooltipV2.Tooltip>
-            {!open && (
+            {!open && !hasOnlySpaces(text) && (
```
Comment (e1himself) [inline packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx:168]: @OskarGasiorowski @aspirisen 

Do you guys think we should additionally disable `/` hotkey reaction when the placeholder is not visible?

Peek 2022-04-27 14-48

**Review by e1himself — APPROVED** [review]
Nice! :+1:

File: packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx (hunk 2 above)
Comment (OskarGasiorowski) [reply packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx:168]: So basically `/` hotkey should work only if a line is empty?

Comment (e1himself) [reply packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx:168]: Maybe, but I'm not sure yet.

@aspirisen what do you think about it?

File: packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx (hunk 1 above)
Comment (aspirisen) [reply packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx:201]: Maybe put it here packages/slate-editor/src/lib ?

File: packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx (hunk 2 above)
Comment (aspirisen) [reply packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx:168]: hm, personally i am not against current behaviour, i would leave it, at least for now

**Comment by aspirisen** [comment]
Shaun The Sheep Movie Ok GIF

**Review by aspirisen — APPROVED** [review]

File: packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx (hunk 1 above)
Comment (OskarGasiorowski) [reply packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx:201]: @aspirisen hmm, let's leave as it is, if we would need in another place we then move it :)

**Pull request merged**